promisasu/Portal
# Update Dependencies

**Taiga User Story:** None

**Taiga Task(s):** None

**What did you change?**
Updated moment, remark and stylelint.
Resolved stylelint deprecation.
Added a few new stylelint rules.

**How can we test?**
1. Run `npm install`
2. Run `npm run lint`
3. No deprecation notices or error messages should appear, some warning messages may appear
